### PR TITLE
scanmem: split package

### PR DESCRIPTION
**Pull request by toluschr**
I made the pr because of this #19098

### CODE REVIEW

**Comment by toluschr** [comment]
I made some changes adding `libscanmem.so.1` to `common/shlibs`

**Comment by toluschr** [comment]
I didn't know, that the shlib file supports automatic SONAME bumps. I removed one extra line.

**Comment by toluschr** [comment]
This should now be ready to be merged.

**Comment by Chocimier** [comment]
Moving gettext-devel to hostmakedepends make cross build fail a bit later.
Mark `nocross="checking for /proc/self/maps... configure: error"`
Please also address xlint check.

**Comment by toluschr** [comment]
I took the license from scanmem's README (https://github.com/scanmem/scanmem#license) and added nocross. I also sorted the hostmakedepends alphabetically.

File: srcpkgs/scanmem/template [hunk 1]
```diff
@@ -1,20 +1,49 @@
...
+
 pre_configure() {
 	NOCONFIGURE=1 ./autogen.sh
 }
+
+libscanmem_package() {
+	short_desc+=" - runtime library"
+	license="LGPL-3.0-only"
+	pkg_install() {
+		vmove usr/lib/*.so*
```
Comment (Chocimier) [inline srcpkgs/scanmem/template]: please quote wildcards in vmove arguments

File: srcpkgs/scanmem/template [hunk 2]
```diff
@@ -1,20 +1,49 @@
...
+	}
+}
+
+gameconqueror_package() {
+	short_desc="GameConqueror is a GUI front-end for scanmem, providing more features"
+	depends="polkit gtk+3 python3-gobject lib${sourcepkg}>=${version}_${revision}"
+	archs=noarch
+	pkg_install() {
+		cd $DESTDIR
+		find * -name '*[Gg]ame[Cc]onqueror*' -prune | while read f; do vmove $f; done
```
Comment (Chocimier) [inline srcpkgs/scanmem/template]: -iname '*gameconqueror*'

**Comment by toluschr** [comment]
I quoted the wildcards properly and changed the line containing find. Tell me if using `*${pkgname}*` instead of `*gameconqueror*` is ok with you.

**Pull request merged**